client/src/amplify-ui/cli-feature-flag/feature-flag.tsx

```diff
@@ -0,0 +1,67 @@
+import {Component, Host, h, Prop, Element} from "@stencil/core";
+import {hostStyle} from "./feature-flag.style";
+import featureFlagsJson from "./feature-flags.json";
+
+type FeatureFlags = Record<string, Section>;
+
+type Section = {
+  description: string;
+  features: Record<string, FeatureFlag>;
+};
+
+export type FeatureFlag = {
+  description: string;
+  type: "Feature" | "Release" | "Experimental";
+  valueType: "Boolean" | "Number" | "String";
+  versionAdded: string;
+  versionDeprecated?: string;
+  deprecationDate?: string;
+  versionRemoved?: string;
+  removalDate?: string;
+  values: Value[];
+};
+
+export type Value = {
+  value: string;
+  description: string;
+  defaultNewProject: boolean;
+  defaultExistingProject: boolean;
+};
```

Comment:
Can we move these into a `.types.ts` file similar to: https://github.com/aws-amplify/docs/blob/master/client/src/docs-ui/page/page.types.ts